README.md

```diff
@@ -1,34 +1,30 @@
-#  *FastForward*
-  [<img src="https://badgen.net/github/checks/FastForwardTeam/FastForward?label=Build" />](https://github.com/FastForwardTeam/FastForward/blob/main/.github/workflows/main.yml)
-  [<img src="https://badgen.net/badge/icon/discord?icon=discord&label=Fast+Forward" />](https://discord.gg/RSAf7b5njt)
-  [<img src="https://img.shields.io/badge/Chromium-Unstable-e77334" />](https://nightly.link/FastForwardTeam/FastForward/workflows/main/main/FastForward_chromium.zip)
-  [<img src="https://img.shields.io/badge/Firefox-Unstable-e77334" />](https://nightly.link/FastForwardTeam/FastForward/workflows/main/main/FastForward_firefox.zip)
- 
+# [<img src="https://i.ibb.co/bW37fgB/Fast-Forward.png" width="50" />](#) _FastForward_
+
+[<img src="https://badgen.net/github/checks/FastForwardTeam/FastForward?label=Build" />](https://github.com/FastForwardTeam/FastForward/blob/main/.github/workflows/main.yml)
+[<img src="https://badgen.net/badge/icon/discord?icon=discord&label=Fast+Forward" />](https://discord.gg/RSAf7b5njt)
+[<img src="https://img.shields.io/badge/Chromium-Unstable-e77334" />](https://nightly.link/FastForwardTeam/FastForward/workflows/main/main/FastForward_chromium.zip)
+[<img src="https://img.shields.io/badge/Firefox-Unstable-e77334" />](https://nightly.link/FastForwardTeam/FastForward/workflows/main/main/FastForward_firefox.zip)
 
 #### Don't waste your time with compliance. FastForward automatically skips annoying link shorteners.
 
-As you may know, universal bypass is no longer maintained by its original developer, Sainan, who had worked on it for so long. 
+As you may know, universal bypass is no longer maintained by its original developer, Sainan, who had worked on it for so long.
 Thanks Sainan for your hard work.
 
-# Installing
-| Jump to section                                     |
-| :-------------                                      |
-| [Chromium based browsers](#chromium-based-browsers) |
-| [Firefox](#firefox)                                 |
+If you are interested in assiting the development or need help, join our Discord server: https://discord.gg/RSAf7b5njt
 
+[<img src="https://i.ibb.co/X70qCQx/Discord-Logo-Color.png" width="40" />](https://discord.gg/RSAf7b5njt)
 
-### Chromium based browsers
+# Installation instructions
 
-Chromium based browsers include Chrome, Edge and Brave 
+## Chromium browsers
```

Comment:
Don't change this, let it be Chromium Based Browsers.

Chrome is a not a Chromium browser.

<sub>also it will break the link on the website sooooo</sub>

Reply:
How do i approve changes?

Reply:
ill watch a vid about this